src/Router.js

```diff
@@ -1,14 +1,18 @@
 import React from 'react';
+import Login from './pages/login/Login';
 import { BrowserRouter, Routes, Route } from 'react-router-dom';
 import Cart from './pages/Cart/Cart';
 import Detail from './pages/Detail/Detail';
 import List from './pages/List/List';
 import Main from './pages/Main/Main';
+import SignUp from './pages/login/SignUp';
 
 const Router = () => {
   return (
     <BrowserRouter>
       <Routes>
+        <Route path="/haimSignUp" element={<SignUp />} />
+        <Route path="/haimLogin" element={<Login />} />
```

Comment:
```suggestion
        <Route path="/signup" element={<SignUp />} />
        <Route path="/login" element={<Login />} />
```
path 이름은 하임Login 처럼 특정 이름이 들어가도록 구현하는건 westagram에서만 적용되는 방법이고,
이 router에서는 signup, logjn 같은 이름으로 수정해주세요!

Reply:
(소중한 haim 잘 보내주고 왔습니다..) 네! 수정 완료했습니다!